crates/turborepo-lib/src/task_graph/mod.rs

```diff
@@ -1,32 +1,15 @@
 mod visitor;
 
-use std::collections::{HashMap, HashSet};
-
 use serde::{Deserialize, Serialize};
 use turbopath::{AnchoredSystemPath, AnchoredSystemPathBuf, RelativeUnixPathBuf};
 pub use visitor::{Error as VisitorError, Visitor};
 
 use crate::{
     cli::OutputLogsMode,
+    config::RawTaskDefinition,
     run::task_id::{TaskId, TaskName},
 };
 
-pub type Pipeline = HashMap<TaskName<'static>, BookkeepingTaskDefinition>;
-
-#[derive(Clone, Debug, Default, Serialize, PartialEq, Eq)]
-pub struct BookkeepingTaskDefinition {
-    pub defined_fields: HashSet<String>,
-    pub experimental_fields: HashSet<String>,
-    pub experimental: TaskDefinitionExperiments,
```

Comment:
we don't have any use cases for this yet, but it was a handy construct for experimental fields that we don't want to end up in `turbo prune` output and also not in dry run / `--summary` output.

Reply:
We can probably do `#[serde(skip)]` for those